code/modules/farming/plant_def/dikarya.dm

```diff
@@ -0,0 +1,147 @@
+/* Cultivars */
+/datum/plant_def/coprinus
```

Comment:
that's not what I meant I mean these should be plant_def/mushroom as they all share vars